book/html.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Constructing a Document Tree
```

Comment:
This is the only place in this chapter using that terminology. Changed to be consistent.